L1Trigger/Phase2L1ParticleFlow/plugins/L1TCorrelatorLayer1Producer.cc

```diff
@@ -175,6 +176,9 @@ L1TCorrelatorLayer1Producer::L1TCorrelatorLayer1Producer(const edm::ParameterSet
 #if 0  // LATER
   produces<l1t::PFCandidateCollection>("TKVtx");
 #endif
+#ifdef PUT_DECODED_TK
+  produces<std::vector<l1t::PFTrack>>("DecodedTK");
+#endif
```

Comment:
why inside `#ifdef` ?

Reply:
@gpetruc this was the decision at the time of the PR: https://github.com/p2l1pfp/cmssw/pull/73. Writing out the decoded tracks is needed only for dedicated studies. That said: I am happy to remove the protection....

Reply:
ok, let's remove it.